scripts/drtprod

```diff
@@ -114,6 +114,32 @@ tags:
 EOF"
 
     case $cluster in
+      "drt-chaos")
+        roachprod ssh ${cluster} -- "sudo mkdir -p /etc/fluent-bit && sudo tee /etc/fluent-bit/config-override.yaml > /dev/null << EOF
+---
+pipeline:
+  inputs:
+  - name: tail
+    path: /var/log/audit/audit.log
+    tag: audit
+    key: message
+    storage.type: filesystem
+    alias: audit
+  outputs:
+  - name: datadog
+    match: audit
+    host: http-intake.logs.${dd_site}
+    tls: on
+    compress: gzip
+    apikey: ${dd_api_key}
+    dd_source: audit
+    dd_service: drt-cockroachdb
+    dd_tags: env:development,cluster:${cluster%:*},service:drt-cockroachdb,team:drt
```

Comment:
Remind me what ${...%:*} means in bash?

Reply:
It removes everything from `:` to the end of the string. For example, given `cluster=drt-large:1` then `${cluster%:*}` will evaluate to `drt-large`. Operators can pass entire clusters (e.g., `drt-large`) or specific hosts (e.g., `drt-large:1`) to `roachprod` commands. This just gives us a clean, but admittedly obscure, way to retrieve the cluster name regardless of how the commands are run.